compose/config/config.py

```diff
@@ -165,13 +164,11 @@ def find_candidates_in_parent_dirs(filenames, path):
     return (candidates, path)
 
 
-@validate_top_level_object
-@validate_service_names
```

Comment:
I moved it into `validate_top_level_object()` so they're always called together now.